src/nimble.nim

```diff
@@ -698,25 +698,31 @@ proc dump(options: Options) =
   echo "backend: ", p.backend.escape
 
 proc init(options: Options) =
-  var nimbleFile: string = ""
-
   if options.forcePrompts != forcePromptYes:
     display("Info:",
       "In order to initialise a new Nimble package, I will need to ask you\n" &
       "some questions. Default values are shown in square brackets, press\n" &
       "enter to use them.", priority = HighPriority)
 
-  # Ask for package name.
+  # Determine the package name.
   let pkgName =
     if options.action.projName != "":
       options.action.projName
     else:
       os.getCurrentDir().splitPath.tail.toValidPackageName()
 
-  nimbleFile = pkgName.changeFileExt("nimble")
-  validatePackageName(nimbleFile.changeFileExt(""))
+  # Validate the package name.
+  validatePackageName(pkgName)
+
+  # Determine the package root.
+  let pkgRoot = if pkgName == os.getCurrentDir().splitPath.tail:
```

Comment:
Small nitpick for future reference: if should be on its own line.

Reply:
Gotcha. I was going to fix it with another pull request to add more licenses but I see that's taken care of already. :)